mtgjson4/compile_mtg.py

```diff
@@ -370,6 +371,9 @@ def build_mtgjson_card(  # pylint: disable=too-many-branches
     if "all_parts" in sf_card:
         mtgjson_card["names"] = []
         for a_part in sf_card["all_parts"]:
+            if "//" in a_part.get("name"):
+                mtgjson_card["names"] = a_part.get("name").split(" // ")
+                break
```

Comment:
Can we drop the `if` condition? A split on a string that isn't contained in the string will return a list of a single element.

Can we strip the parts instead of splitting with slashes and whitespace?

For example, we could collapse this to:

```python
if "all_parts" in sf_card:
    mtgjson_card["names"] = []
    for card_part in sf_card["all_parts"]:
        mtgjson_card["names"].extend(name.strip() for name in card_part.get("name", "").split("//"))
```